### ISSUE TITLE
Stuck in the void upon respawning with a "Tried teleporting you to an unloaded ship" message in chat

### ISSUE THREAD

**Issue by hoobooh** [ValkyrienSkies/Valkyrien-Skies#345]
No mods that may affect spawning other than Waystones

**Comment by AwesomeAnimationGamer** [comment]
I'm getting this too

**Comment by AwesomeAnimationGamer** [comment]
Some more information: I was, at the time, running no mods other than VS. I placed a bed, slept, then jumped off the ship and let myself fall to my death. Upon respawning, I got the "Tried teleporting you to an unloaded ship" and was in the void. I manually teleported to the coordinates the ship was supposed to be at, and I was back in the overworld. Additionally, the ship was still there. However, it no longer had collisions, it was like a ghost ship. I could pass straight through it, couldn't build or break blocks, and most importantly, I couldn't interact with the Physics Infuser to disassemble it. This was on build 137, I haven't tested in Master yet.
